streams/src/main/java/org/apache/kafka/streams/kstream/internals/TimestampedKeyValueStoreMaterializer.java

```diff
@@ -33,7 +32,7 @@ public TimestampedKeyValueStoreMaterializer(final MaterializedInternal<K, V, Key
     /**
      * @return  StoreBuilder
      */
-    public StoreBuilder<TimestampedKeyValueStore<K, V>> materialize() {
+    public StoreBuilder<?> materialize() {
```

Comment:
Why can't we keep the generic typer here? I understand that we need to make it `<?>` in `KTableImpl` but not sure why we need `<?>` here, too?

Reply:
I we need it later, it's fine to do it right away.